src/codegen/doc/mod.rs

```diff
@@ -503,7 +513,28 @@ where
             )?;
         }
 
-        for parameter in fn_.parameters() {
+        let parameters: Vec<_> = fn_
+            .parameters()
+            .iter()
+            .filter(|param| {
+                let is_array_length_param = if param.name.len() > 3 {
+                    let array_param_name = &param.name[2..];
+                    &param.name[0..2] == "n_"
+                        && fn_.parameters().iter().any(|p| p.name == array_param_name)
+                } else {
+                    false
+                };
```

Comment:
```suggestion
                let is_array_length_param = if let Some(array_param_name) = param.name.strip_prefix("n_) {                    
                    fn_.parameters().iter().any(|p| p.name == array_param_name)
                } else {
                    false
                };
```

Reply:
Ah that looks much better. TIL a bout strip_prefix. Thanks!